app/assets/stylesheets/taxs.scss

```diff
@@ -1,467 +1,930 @@
-@import 'govuk/_grid_layout.scss';
-@import 'govuk/_conditionals.scss';
-@import 'govuk/_colours.scss';
-@import 'govuk/_typography.scss';
-@import 'govuk/_helpers.scss';
-@import 'govuk/_panels.scss';
-@import 'govuk/design-patterns/_buttons.scss';
-@import 'govuk/design-patterns/_alpha-beta.scss';
-@import 'bar-chart/_bar-chart.scss';
+@import 'partials/_palette';
```

Comment:
I've had a scan through this CSS. It looks like it makes sense but is it all needed?

Reply:
I just coped it from https://gist.github.com/adamliptrot-oc/f77250a6f69fb31fabd935e2002f4964
I think its all needed right @adamliptrot-oc ?

Reply:
The palette and focus files are both needed in their entirety (both for support and for ease of upgrading).
They should be added after all other css though and it looks like there is a lot of other css being loaded after them here which could mean these get overridden.
I'd also suggest putting the service-specific stuff into individual scss files to better organise them.

Reply:
done

Reply:
Sorry, meant the file I was commenting on rather than the one the line was on. Should have been clearer